src/po/ja.po

```diff
@@ -205,6 +208,33 @@ msgstr "    行=%ld  識別子=%d  名前=%s"
 msgid "E902: Cannot connect to port"
 msgstr "E902: ポートに接続できません"
 
+msgid "E901: gethostbyname() in channel_open()"
+msgstr "E901: channel_open() 内の gethostbyname() が失敗しました"
+
+msgid "E898: socket() in channel_open()"
+msgstr "E898: channel_open() 内の socket() が失敗しました"
+
+msgid "E903: received command with non-string argument"
+msgstr "E903: 非文字列の引数のコマンドを受信しました"
+
+msgid "E904: last argument for expr/call must be a number"
+msgstr "E904: expr/call の最後の引数は数字でなければなりません"
+
+msgid "E904: third argument for call must be a list"
+msgstr "E686: call の3番めの引数はリスト型でなければなりません"
```

Comment:
@crazymaster thank you and fixed. https://github.com/vim-jp/lang-ja/commit/1ad189603b91f658f5a01f9e608b33a598b932bf